scala/build/Dockerfile.mantic

```
@@ -27,7 +27,7 @@ ENV PATH="/venv/bin:$PATH"
 RUN pip install --upgrade pip
 
 # Copy the requirements file into the container
-COPY scala/requirements.txt .
+COPY scala/build/requirements.txt .
```

Hadolint

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)